### PR TITLE
Isomphoric String works fine

### PR DESCRIPTION
**Pull request by noor848**
image

### CODE REVIEW

**Comment by hayatzsala** [comment]
Target branch update

File: Problem.sln [hunk 1]
```diff
@@ -0,0 +1,25 @@
+﻿
```
Comment (hayatzsala) [inline Problem.sln:1]: same as project 3, add it to the solution

File: Prob4/Program.cs [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        return true;
+
+
+
+
+
+
+
+    }
+}
```
Comment (hayatzsala) [inline Prob4/Program.cs:47]: @noor848 Main method ?

File: Prob4/Program.cs [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+            if (!charcters.ContainsKey(s[i]))
+                charcters.Add(s[i], t[i]);
+        }
+
+
+
+        for (int i = 0; i < s.Length; i++)
+        {
+            char value;
+            charcters.TryGetValue(s[i], out value);
```
Comment (hayatzsala) [inline Prob4/Program.cs:15]: TryGetValue ❤️

File: Prob4/Program.cs [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        Dictionary<char, char> c2 = new Dictionary<char, char>();
+
+        for (int i = 0; i < t.Length; i++)
+        {
+            if (!c2.ContainsKey(t[i]))
+                c2.Add(t[i], s[i]);
+        }
+
+
+        bool vv = true;
```
Comment (hayatzsala) [inline Prob4/Program.cs]: unused variable, vv

File: Prob4/Program.cs (hunk 2 above)
Comment (noor848) [reply Prob4/Program.cs:47]: Fixed

**Review by hayatzsala — APPROVED** [review]

**Pull request merged**